include/flatbuffers/flexbuffers.h

```diff
@@ -368,6 +368,10 @@ class Reference {
   bool IsString() const { return type_ == FBT_STRING; }
   bool IsKey() const { return type_ == FBT_KEY; }
   bool IsVector() const { return type_ == FBT_VECTOR || type_ == FBT_MAP; }
+  bool IsTypedVector() const { return flexbuffers::IsTypedVector(type_); }
+  bool IsTypedVectorElementType() const { return flexbuffers::IsTypedVectorElementType(type_); }
```

Comment:
`IsTypedVectorElementType` doesn't seem of concern to the user most of the time?

Reply:
don't know.. not useful for me at this time. removing it completely. can add that separately if clients need it.